content/03-reference/01-tools-and-interfaces/02-prisma-client/05-filtering.mdx

```diff
@@ -0,0 +1,323 @@
+---
+title: 'Filtering'
+metaTitle: 'Filtering (Reference)'
+metaDescription: 'Advanced filtering use cases.'
+---
+
+## Overview
+
+Prisma Client allows you to filter records on any combination of model fields, including related models, and supports a variety of filter conditions.
+
+For example, the following query returns all `User` records with at least one published post and an email address that ends with `prisma.io`:
+
+```javascript
+const filteredUsers = await prisma.user.findMany({
+  where: {
+    email: {
+      endsWith: 'prisma.io',
+    },
+    posts: {
+      some: {
+        published: true,
+      },
+    },
+  },
+})
+```
+
+The following queries support filtering:
+
+- `findMany`
+- `updateMany`
+- `deleteMany`
+
+## Filter conditions
+
+Prisma Client supports the following filter conditions:
+
+| Operator     | Notes                                                |
```

Comment:
Are all operators available for all available types (e.g. can you use `startsWith` on `Int`s)? If no, it be worth adding another column that shows to which types an operator applies.

Reply:
Done! ✅